examples/with-apollo/src/client-components/apollo-client-provider.tsx

```diff
@@ -0,0 +1,16 @@
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
+
+export const client = new ApolloClient({
+  // URL retrieved from official apollo graphql blog
+  // See https://www.apollographql.com/blog/using-apollo-client-with-next-js-13-releasing-an-official-library-to-support-the-app-router
+  uri: "https://main--spacex-l4uc6p.apollographos.net/graphql",
```

Comment:
Proposal: It'd be nice if we open a graphql API endpoint like the [ones already used on tests](https://next-data-api-endpoint.vercel.app/api/random).

Reply:
@devjiwonchoi Definitely a nice-to-have! But not strictly required for `/examples`.

Reply:
Yeah, IMO it'd be great to prevent issues like: https://github.com/vercel/next.js/issues/36112#issuecomment-1098849154

Thanks for reviewing!!